.travis.yml

```diff
@@ -10,6 +10,7 @@ rvm:
  - 2.5
 branches:
  only:
+  - feature-53943-e2e-ci-job
   - stable
```

Comment:
Adding this line temporarily, to trigger a build on this branch on travis and test whether the manual build does execute the system tests.